.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    - deploy
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build containers and run tests
+      run: docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm scraper
```

Comment:
generally, we don't run our tests within containers on CI. 

instead we set up the environments and such locally and run them that way.

this is a good pattern to follow: https://github.com/datamade/django-councilmatic/blob/5b074f376667766e4b6dbf093871d294bb35fc51/.github/workflows/main.yml#L32-L48

this will also help you set the cache path to what's expected.

Reply:
this test is passing now, but other tests that address parts of the code unrelated to the work I've been doing so far. I'm unsure how to proceed here.

Reply:
relatedly, what's the best way for me to run tests locally, then? I don't think I've run tests locally without containers; should I keep the `tests/docker-compose.yml` file I had in order to run tests locally? I'm thinking that could help keep environments consistent.